### ISSUE TITLE
div class 'flatpickr-calendar' not getting removed when changing route in vue-router

### ISSUE THREAD

**Issue by diffused** [jrainlau/vue-flatpickr#53]
I've got an SPA application that uses vue-router in the standard manner. Named routes, vue components, etc.

vue-flatpickr has been setup in my main.js as such:
```import VueFlatpickr from 'vue-flatpickr'
import 'vue-flatpickr/theme/airbnb.css'
Vue.use(VueFlatpickr)```

When a page/route is loaded with a vue-flatpickr, a hidden calendar div is added to the bottom of the html as expected.
```<div class="flatpickr-calendar">...</div>```

When I navigate away to another page/route, the calendar div doesn't get destroyed. It just lives on. Subsequent navigations back to the page/route with the calendar append a new calendar div. Multiple navigations will stack many calendar divs.

My app is built upon the vue-cli webpack project.

My dependencies are:
```"dependencies": {
    "bulma": "^0.3.1",
    "js-cookie": "^2.1.3",
    "mini-toastr": "^0.3.10",
    "numeral": "^2.0.4",
    "vee-validate": "^2.0.0-beta.18",
    "vue": "^2.1.0",
    "vue-flatpickr": "^2.2.9",
    "vue-notifications": "^0.7.6",
    "vue-resource": "^1.0.3",
    "vue-router": "^2.1.1"
  }```

I just did a quick experiment and added [vue-flatpickr.vue](https://github.com/jrainlau/vue-flatpickr/blob/2.0/src/components/vue-flatpickr.vue) directly into my project, and imported it into my page component. It does seem to remove the calendar div during navigations. 

Any ideas?

**Comment by CalebKester** [comment]
Ran into this same issue. Wondering what the fix is.

**Comment by arlando** [comment]
This issue will also occur if you are using vue-flatpickr inside of a modal.

I was investigating this issue and noticed a few weird things.

- the vue-flatpickr code in NPM seems to be missing this critical line: https://github.com/jrainlau/vue-flatpickr/blob/2.0/src/components/vue-flatpickr.vue#L58 I added it back to the dist/index.js and the element started removing itself, there was a pull request for this issue a while back: https://github.com/jrainlau/vue-flatpickr/commit/1927ade1001fcf46a0b6b76107dd4d28d384e40b. It was pulled into 2.0 branch on Jan 31 2017 

- weird thing is that NPM thinks this library is on version 2.3.0, where as 2.2.9 is referenced as the package JSON. I tried grabbing 2.2.9 directly and it has the same bug dist/index.js

- using `$ npm view vue-flatpickr` the release of version 2.3.0 happened on 2017-02-10, but the dist/index.js does not have the fix, the other latest release in 2.2.9 released on Jan 20th is impossible to have the fix because the fix wasn't pulled into the 2.0 branch yet...

Long story short @jrainlau can you rebuild the dist/index.js to include the pull request mentioned above and push it to NPM under a new version. Also update the version in package.json to the right one? Thanks!

In the mean time I managed to fix the bug by first caching the flatpickr reference via an event binding and in the Vue component's destroy method, calling destroy on the flatpickr instance.
